test/08-tasks.test.ts

```diff
@@ -408,5 +408,44 @@ describe('08 task tests', function () {
 				})
 				.expect(500);
 		});
+
+		it('should not pick up new task after worker is stopped', async () => {
```

Comment:
Can we also add a test that ensures that a stopped worker can then be started again with `worker.start()` and have it pick back up where it stopped? Just to make sure things make sense from the consumer side and work as expected.

Reply:
So I wanted to add a separate test for it, however, the more interesting test IMHO is a test that picks up a task that was created before the worker is stopped and then picked up. To break this down into two tests I would have to forward the task id and all other parameters from one test to another, which I didn't like. So I ended up with one test that stops the worker, test that it is not picked up, starts it and ensure the tasks runs.